CHANGELOG.md

```diff
@@ -29,6 +31,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 This is the initial release to crates.io.
 
-[Unreleased]: https://github.com/rust-embedded-community/usbd-serial/compare/v0.2.0...HEAD
+[Unreleased]: https://github.com/rust-embedded-community/usbd-serial/compare/v0.2.1...HEAD
+[0.2.1]: https://github.com/rust-embedded-community/usbd-serial/compare/v0.2.0...v0.2.0
```

Comment:
```suggestion
[0.2.1]: https://github.com/rust-embedded-community/usbd-serial/compare/v0.2.0...v0.2.1
```

Reply:
Perhaps it would be better to link to the Github Release page, where it is fairly trivial to show the diff compared to the previous release. Saves calculating these URLs by hand.

Reply:
Hmm good point. I've updated the ones that have a Github release